game/plugins/src/main/kotlin/gg/rsmod/plugins/content/combat/strategy/MagicCombatStrategy.kt

```diff
@@ -64,6 +64,28 @@ object MagicCombatStrategy : CombatStrategy {
         if (pawn is Player) {
             MagicSpells.getMetadata(spell.uniqueId)
                 ?.let { requirement -> MagicSpells.removeRunes(pawn, requirement.runes) }
+            /* ------- SPELL SOUNDS -------- */
```

Comment:
It's better to put the sounds into the spells enum, rather than having multiple lines like this.

`pawn.playSound(spell.sound)`

Reply:
resolved